std/conv.d

```diff
@@ -3737,10 +3737,21 @@ unittest
 @property T octal(T, string num)()
     if (isOctalLiteral!num)
 {
+    return octal!T(num);
+}
+
+/// Ditto
+@property T octal(T)(const(char)[] num) pure nothrow
```

Comment:
Do you mean range as in `InputRange`/`ForwardRange` or as in `char`/`wchar`/`dchar`?